README.adoc

```diff
@@ -382,6 +382,10 @@ In addition to the macros listed above that can be used only inside `lutaml_uml_
 [source,adoc]
 -----
 This is lutaml_figure::[package="Wrapper root package", name="Fig B1 Full model"] figure
+This is lutaml_table_package::[package="Wrapper root package"] package
+This is lutaml_table_class::[package="Wrapper root package", name="my name"] class
+This is lutaml_table_enum::[package="Wrapper root package", name="my name"] enum
+This is lutaml_table_data_type::[package="Wrapper root package", name="my name"] data_type
```

Comment:
Can we simplify these commands into:
* `lutaml_uml_table::[package="foo"]` 
* `lutaml_uml_table::[package="foo",class="bar"]`
* `lutaml_uml_table::[package="foo",enum="bar"]`
* `lutaml_uml_table::[package="foo",datatype="bar"]`
?

Reply:
yes, sure